packages/page-claims/src/Statement.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017-2020 @polkadot/app-claims authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { StatementKind } from '@polkadot/types/interfaces';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import RegularStatement from './statements/regular';
+import SaftStatement from './statements/saft';
+import { useTranslation } from './translate';
+import { getStatementUrl } from './util';
+
+export interface Props {
+  className?: string;
+  kind: StatementKind;
+}
+
+function Statement ({ className, kind }: Props): React.ReactElement<Props> | null {
+  const { t } = useTranslation();
+  const statementUrl = getStatementUrl(kind);
+
+  return (
+    <div className={className}>
+      {t('Please read these terms and conditions carefully. By submitting this statement, you are deemed to have accepted these Terms and Conditions. If you do not agree to these terms, please refrain from accessing or proceeding. You can also find them at: ')}
```

Comment:
Spacing. Remove the end-space, if you need a space, rather use a style to add a margin.